src/NuProj.ProjectSystem.12/Rules/general.xaml

```diff
@@ -1,33 +1,46 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry for this - it seems I have an extension that automatically reformats xaml files on save. I will revert such files in the pull request and re-commit shortly